subtests/docker_cli/dockerinspect/inspect_keys.py

```diff
@@ -43,13 +43,30 @@ def run_once(self):
         json = self.inspect_and_parse(subargs)
         self.sub_stuff['image_config'] = self.filter_keys(json)
 
-    @staticmethod
-    def filter_keys(json):
+    def filter_keys(self, json):
+        json0 = json[0]
         # Keys in Labels should not be checked for style
         try:
-            del json[0]['Config']['Labels']
+            json0['Config']['Labels'] = {}
+            self.logdebug("Cleared Config.Labels branch")
         except KeyError:
-            pass
+            self.logdebug("Config.Labels branch not found?")
+        # These are data-keys, not structural, ignore names
+        try:
+            # New in docker 1.9.1
+            networks = json0['NetworkSettings']['Networks']
+            for idx, network in enumerate(networks.keys()):
+                # Child key names still need checking
+                dummy_key = 'NameIgnored%d' % idx
```

Comment:
Oh whew, thought I lost something.  What I've found easiest, is to clone down my forked branch (the source of this PR), mangle^H^H^H^Hfix it however you like, then send it to me as a PR (against my personal fork/branch).  Hint: It's the drop-downs on the top-left in the 'compare' screen, not the ones on the left :grinning: